Remove myself from assignees

kamack38 opened the pull request:
I didn't have a lot of time and I probably will not have enough time in the near future to maintain this project, so for the time being, I'm removing myself from the assignees to every issue.

ldrahnik commented:
@kamack38 Np. What about arch package? What is your opinion? Could you please look at reported issues? I do not have access there neither I do not use arch. Does not make sense to me provide / link to not working arch package. Keep up-to-date only one source git repository sounds better to me.

kamack38 commented:
> @kamack38 Np. Thank you for your contributing. What about arch package? What is your opinion? Could you please look at reported issues? I do not have access there neither I do not use arch. Does not make sense to me provide / link to not working arch package. Keep up-to-date only one source git repository sounds better to me.

I've fixed the issue that you talked about but it wasn't pushed because the package didn't build. I've fixed that today so I think it should work now.
